### PR TITLE
Add demo of recording audio debrief

### PR DESCRIPTION
**Pull request by mathieuhendey**

### CODE REVIEW

File: audio-debrief-recording/src/pages/home/home.ts [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  }
+
+  logError(error) {
+    console.log(error);
+  }
+
+  stopRecordingAudio() {
+    console.log("Stopping recording");
+
+    var self = this;
```
Comment (danielchildsbjss) [inline audio-debrief-recording/src/pages/home/home.ts:66]: Do we still need to do this in typescript?

File: audio-debrief-recording/src/pages/home/home.ts [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    var timerDur = setInterval(() => {
+      // workaround we can't stop playback just after call .play(). We trying to do it in interval...
+      this.audio.pause();
+      this.audio.stop();
+      counter = counter + 100;
+      if (counter > 2000) {
+        clearInterval(timerDur);
+      }
+      this.audio.getDuration()
+      var dur = this.audio.getDuration();
```
Comment (danielchildsbjss) [inline audio-debrief-recording/src/pages/home/home.ts:99]: Do we have to getDuration twice?

File: audio-debrief-recording/src/pages/home/home.ts [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  recordButtonText = 'Record audio';
+  startingRecording = true;
+  playButtonText = 'Play';
+
+  recordAudio() {
+    if (this.startingRecording) {
+      this.file.createFile(this.file.dataDirectory, this.fileName, true).then(() => {
+        let src = this.file.dataDirectory.replace(/^file:\/\//, '') + this.fileName;
+        // ionic media.create is buggy (getDuration returns always -1), the solution is to use raw cordova API
+        this.audio = new (<any>window).Media(src, this.mediaSuccess, this.mediaError, this.mediaStatus);
```
Comment (danielchildsbjss) [inline audio-debrief-recording/src/pages/home/home.ts:31]: any type file for this?

**Review by danielchildsbjss — APPROVED** [review]
Looks fine. I'm a bit concerned about the getDuration problem though, but it looks like a problem with other libraries too - https://stackoverflow.com/a/7052147. 
Maybe we can get away with estimating it based on file size.

I'd also like us to move the logic out into services when we use this in the main application.

File: audio-debrief-recording/src/pages/home/home.ts (hunk 3 above)
Comment (zielinskikamil) [reply audio-debrief-recording/src/pages/home/home.ts:31]: we had to use here Cordova API and unfortunately, there are no TS typings for it. (Ionic wrapper wasn't working here - getDuration() method was not working)
If you check the **this.audio** type you will see that actually we are using Ionic media type (as it has the same API but is not the same 'class')

**Pull request merged**